src/versionbits.h

```diff
@@ -24,10 +24,10 @@ static const int32_t VERSIONBITS_NUM_BITS = 29;
  */
 enum class ThresholdState {
     DEFINED,   // First state that each softfork starts out as. The genesis block is by definition in this state for each deployment.
-    STARTED,   // For blocks past the starttime.
+    STARTED,   // For blocks past the startheight.
     LOCKED_IN, // For one retarget period after the first retarget period with STARTED blocks of which at least threshold have the associated bit set in nVersion.
     ACTIVE,    // For all blocks after the LOCKED_IN retarget period (final state)
```

Comment:
I think you should document "For all blocks after the LOCKED_IN retarget period, if min activation height has been reached" ?

Reply:
Done